graphene_sqlalchemy/registry.py

```diff
@@ -1,32 +1,89 @@
+from sqlalchemy.types import Enum as SQLAlchemyEnumType
+
+from graphene import Enum
+
+
 class Registry(object):
     def __init__(self):
         self._registry = {}
         self._registry_models = {}
+        self._registry_orm_fields = {}
```

Comment:
```suggestion
        self._registry_orm_fields = defaultdict(dict)
```

If you make this a defaultdict, you don't have to handle missing object types below.

Reply:
I also thought about that but was not sure what was simpler. But ok, let's use the batteries included.